nav2_gps_waypoint_follower_demo/models/turtlebot_with_gps/model.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+
+<model>
```

Comment:
This file... contains no model? Does this need to behere?

Reply:
I think this file should be included in the same folder as the model itself to encode the model metadata, as suggested by [gazebo docs](https://classic.gazebosim.org/tutorials?tut=model_structure). Also, this is the way the original turtlebot models seem to be made, ex: the [turtlebot waffle model used by nav2](https://github.com/ROBOTIS-GIT/turtlebot3_simulations/tree/ros2/turtlebot3_gazebo/models/turtlebot3_waffle).

I tried removing it but got an error from gazebo's parser

However let me know if you are aware of another way this can be moved / removed / replaced